viz/charts/linkerd-viz/templates/tap-policy.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: Server
+metadata:
+  namespace: {{.Values.namespace}}
+  name: tap-admin
+  labels:
+    linkerd.io/extension: viz
+    component: tap
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+spec:
+  podSelector:
+    matchLabels:
+      linkerd.io/extension: viz
+      component: tap
+  port: admin-http
```

Comment:
tioli: we could probably have a single server policy for all ports named `admin-http` in this namespace; not a blocker though